src/Microsoft.Health.Fhir.Core/Features/Validation/AttributeValidator.cs

```diff
@@ -41,7 +42,10 @@ public IEnumerable<ValidationFailure> Validate(PropertyValidatorContext context)
                 {
                     foreach (var error in results)
                     {
-                        yield return new ValidationFailure(error.MemberNames?.FirstOrDefault(), error.ErrorMessage);
+                        ITypedElement typedElement = resourceElement.Instance;
+                        var fullFhirPath = typedElement.InstanceType + "." + error.MemberNames?.FirstOrDefault();
```

Comment:
It's not entirely obvious that this works for paths that are deeper than top-level fields, but from your screenshot it looks like it does. :) Could you add a unit test to verify the path?

Reply:
Yep definitely!

Reply:
This might not be a problem right now since we currently have recursion set to false: 
https://github.com/microsoft/fhir-server/blob/master/src/Microsoft.Health.Fhir.Core/Features/Validation/AttributeValidator.cs#L40

Reply:
Let me know what you think of cf799da and if you think it makes sense to add new unit tests for AttributeValidator.

Reply:
Added an AttributeValidator unit test in cc717d1.